### DISCUSSION TITLE
Buffs - Group Only

### DISCUSSION THREAD

**Discussion by Encrypt512bit** [smasherprog/EqTool#24] [Ideas]
Nparse is able to make it so it captures only group member buffs, is it possible to add a setting for this? I find it annoying sometimes when I'm in a dungeon group and I catch buffs from other PCs around me.

**Comment by smasherprog** [comment]
So, in order to accomplish this, NParse ONLY shows spells that are cast by you only. I could add that as an option. If you only want to see buffs that you cast and no others, thats a possibility. I just remove the people from the buff list.

<img width="239" alt="image" src="https://github.com/smasherprog/EqTool/assets/3393733/9d1f9ac3-5e74-4981-adb1-c109cf5d991c">

**Comment by smasherprog** [comment]
if you click that it removes the person entirely.

Come to think of it, thats probably the right way to go in this scenario.

**Comment by smasherprog** [comment]
I try to capture the 90/10 case. and most of the time this is a non issue, and when it is an issue the work around is very simple.   Often times you have to remove npcs from the list because their death isnt capture in the log file. So they stay in the list until their timers run off

**Comment by smasherprog** [comment]
What do you think? Id like to see some good discussion here so please jump in!

**Reply by Encrypt512bit** [reply]
Having an option to only capture buffs you cast would be cool. Sometimes capturing everything is really nice. Other times it can just get crowded.
